charts/galoy/templates/admin-deployment.yaml

```diff
@@ -98,8 +98,13 @@ spec:
         - name: KRATOS_MASTER_PHONE_PASSWORD
           valueFrom:
             secretKeyRef:
-              name: {{ .Values.galoy.kratos.masterPhonePasswordExistingSecret.name }}
-              key: {{ .Values.galoy.kratos.masterPhonePasswordExistingSecret.key }}
+              name: {{ .Values.galoy.kratos.existingSecret.name }}
+              key: {{ .Values.galoy.kratos.existingSecret.password }}
+        - name: KRATOS_API_KEY
```

Comment:
```suggestion
        - name: KRATOS_CALLBACK_API_KEY
```

Reply:
it's better to be more precise about that this API key is about. it's only to protect the callback api on registration.

Reply:
Fixed in 61d499f3f069174db0833c3d86bdaa2941074fff